### ISSUE TITLE
cannot visualize images in debug console while debugging

### ISSUE THREAD

**Issue by seyeeet** [microsoft/vscode-remote-release#4979]
I have being dealing with this issue for weeks now, please help :(
I am using vscode on my local system (windows), then I connect to a head node (linux ) and after requesting a node (it is slurm) then run my job on the selected node (e.g. node1).  so I need to do:
`ssh usr@headip` and then after doing `salloc ....` to get a node, I need to do `ssh node1` to go to that node.
Now my issue is that I cannot `display` any images or visualize anything when I am **debugging**, I can display stuff when I am not debugging though.

I tried so many things starting from installing the X11 forwarding to setting the display and etc. I should clarify that I dont want to visualize stuff out of the debugging console and I like to see them inside that.
For example when I am debugging, I like to see the image in the `debug console` when I do `plt.imshow(img)`.
basically there is no output for  `plt.imshow(img)`.

Can some one please help ....

- VSCode Version: 1.55.2 (system setup)
- Local OS Version: windows 10
- Remote OS Version: it is a cluster and then nodes (but everything is linux)
- Remote Extension/Connection Type: SSH
- Logs: not sure

**Comment by roblourens** [comment]
> I can display stuff when I am not debugging though.

Not clear what you mean here. This is not supported in the debug console at all, to my knowledge, ssh or not

**Comment by seyeeet** [comment]
@roblourens can it become supported?

**Comment by roblourens** [comment]
No, we don't plan to support custom visualizations like that inside the debug console.

To get that feature in the linked issue to work, you could set up X11 forwarding, if you are connecting to a linux remote. You will have to configure that yourself.
